prettier.config.ts

```diff
@@ -4,7 +4,8 @@ const options: Options = {
   trailingComma: 'none',
   semi: false,
   singleQuote: true,
-  proseWrap: 'always'
+  proseWrap: 'always',
+  printWidth: 130
```

Comment:
What about only adjusting this for markdown files? https://prettier.io/docs/en/configuration.html#configuration-overrides

Reply:
Yes, I initially thought this is the way to do it, but...
`proseWrap` affects `.md` files only from what I understand
`printWidth` configures `proseWrap`

hence

no need to add an override, but rather another option that changes the default behavior of `proseWrap`